.buildkite/metricbeat/pipeline.yml

```diff
@@ -107,6 +107,7 @@ steps:
           set -euo pipefail
           echo "~~~ Running tests"
           export KUBECONFIG="$$PWD/kubecfg"
+          asdf install kind "${ASDF_KIND_VERSION}"
```

Comment:
Installing this at runtime defeats the purpose of pre-fetching the binaries.

Can you please address this https://github.com/elastic/ci-agent-images/blob/main/vm-images/platform-ingest/scripts/beats/kind.sh#L5C1-L5C27 and then bump the version without runtime installation?

I can help with running the packaging of new VM images.

Reply:
Ah, that’s why I couldn’t find any references to version v0.20.0 of Kind in Beats—it’s inherited from the CI images. It all makes sense now, thanks!

Do we still need to set ASDF_KIND_VERSION in the pipeline config if the version is already preinstalled? It doesn’t seem necessary if it’s defined in the CI images instead, and it only adds confusion when it’s time to update them.

I'll open a PR to bump the version in the ci-agent-images repo.

Reply:
I have opened https://github.com/elastic/ci-agent-images/pull/991. I'd appreciate the assistance with the VM images once this is merged. Thanks!

Reply:
Now we can just do the env var bump without the `asdf install kind...`

Reply:
Oh, that looks pretty simple. Don't we have to build the ci-images first or that is done automatically?

Reply:
Seems like the tests are still failing with the error, I see that kind is [still 0.20.0 in the image](https://buildkite.com/elastic/beats-metricbeat/builds/10767#0192b98f-7bc5-4c8f-b6e5-bc4db92b54a8/146-152). Should I bump the image version somewhere? Thanks.

Reply:
>Oh, that looks pretty simple. Don't we have to build/bump the ci-images first or that is done automatically?

There is already a discussion about how to handle the version bumps best.
For now, it requires some extra work.

I can generate new images that would include the latest change (version 0.24), but then we would need to follow up with merging this, because it will prevent other CI runs.
Let me think about this and get back to you, we might need to install 0.20 and 0.24 so we don't break anything.

Reply:
raised https://github.com/elastic/ci-agent-images/pull/994 to fix this issue.